### ISSUE TITLE
Why `System.Runtime.Caching` has dependency to `System.Security.Principal.Windows`?

### ISSUE THREAD

**Issue by dehghani-mehdi** [dotnet/corefx#42765]
Why `System.Runtime.Caching` has dependency to `System.Security.Principal.Windows`?

**Comment by stephentoub** [comment]
@dehghani-mehdi, why did you open a new issue here?  https://github.com/dotnet/corefx/issues/42592 had been tracking your previous same question, and it was answered there.  You also replied to that one saying it could be closed (though your response appears to have been deleted).

**Comment by dehghani-mehdi** [comment]
Because it's not _and that was not_ a question, it's a bug (maybe I should change the title to something like `System.Runtime.Caching` should not has dependency to `System.Security.Principal.Windows`). you wrote some technical stuff there, for my as user the reason is not matter, the issue is why memory cache has dependency to some unrelated package and why I have to include some un-related dll in my project.

**Comment by stephentoub** [comment]
> Because it's not

Maybe it was a misunderstanding.  @StephenBonikowsky asked whether the issue needed to remain open, and you responded "no".  But maybe you were answering a different question.

> Yes, because you closed the issue.

I closed the issue because you commented that no issues remained.  It sounds like we had a miscommunication.

> it's a bug

What's the bug?  Are you saying that something is breaking at runtime?  Or you just don't want this extra .dll deployed as part of your app?  @vcsjones outlined nicely in https://github.com/dotnet/corefx/issues/42592#issuecomment-553989245 why this dependency is there.

**Comment by dehghani-mehdi** [comment]
> I closed the issue because you commented that the issue needn't be kept open

I meant the question _and the issue_ was not answered _solved._ but that comment. _for the record, the comment was: 'I don't think so'_

>  It sounds like we had a miscommunication.

Yes, it seems like we had. sorry I didn't explain well.

> Are you saying that something is breaking at runtime?

No, everything works great

> Or you just don't want this extra .dll deployed as part of your app? 

Yes, I am developer too, it is clear (at least to me) memory cache has nothing to do with `System.Security.Principal.Windows`, I'm Windows user, but the name `Windows` here is another problem, It seems like hard-coded value to me.
But my main issue is that extra dll.

**Comment by StephenMolloy** [comment]
I don't believe this is a System.Runtime.Caching bug at this point. The cache package relies on ConfigurationManager, and that is expected. The issue that remains is that System.Security.Principal.Windows does not get linked away as expected, because of System.Net.Sockets? https://github.com/dotnet/corefx/issues/42592#issuecomment-555680784

**Comment by danmosemsft** [comment]
Updating title to match my reading of that comment.
